sys/include/phydat.h

```diff
@@ -107,7 +107,8 @@ enum {
     UNIT_PPM,       /**< part per million */
     /* aggregate values */
     UNIT_TIME,      /**< the three dimensions contain sec, min, and hours */
-    UNIT_DATE       /**< the 3 dimensions contain days, months and years */
+    UNIT_DATE,      /**< the 3 dimensions contain days, months and years */
+    UNIT_PWM        /**< the 3 dimensions contain duty cycle, frequency and resolution */
```

Comment:
this does IMHO not make any sense. PWM is not a physical value in itself, so I don't see it as part of this list. What we do control when using a PWM is actually the duty cycle, so why not use `UNIT_PERMILL` as unit for the SAUL pwm driver?